src/main/java/com/nutomic/syncthingandroid/fragments/FolderFragment.java

```diff
@@ -76,7 +77,7 @@
 
     private TextView mVersioningKeepView;
 
-    private boolean mIsCreate;
+    private boolean mCreateMode;
```

Comment:
I think `mIsCreateMode` is better for a boolean ("is" showing that it's either true or false). But not really important.